### PR TITLE
Migrate documentation to Laika (sbt-typelevel-site)

### PR DESCRIPTION
**Pull request by CJSmith-0141**
addresses #14 

edit: Adding some screenshots

Landing page 
Screenshot 2024-03-22 at 11 32 46 AM

Sample report
Screenshot 2024-03-22 at 11 47 56 AM

Another sample report
Screenshot 2024-03-22 at 11 51 57 AM

Expectations page
127 0 0 1_4242_features_expectations html

### CODE REVIEW

**Comment by CJSmith-0141** [comment]
going to re open this and be more careful with my force push in the future lol

File: docs/assets/sample_expectation_suite_output.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+}
+div.terminal pre {
+  background-color: black;
+  color: white;
+  padding: 20px;
+  overflow: auto
+}
+</style>
+<body>
+```scala mdoc:passthrough
```
Comment (Baccata) [inline docs/assets/sample_expectation_suite_output.html:17]: I take it this is where you've struggled ? Is Laika somehow able to run this snippet via a call to mdoc ?

Comment (CJSmith-0141) [reply docs/assets/sample_expectation_suite_output.html:17]: When this passthrough is in a markdown document, Laika hangs. Mdoc doesn't have an issue. By moving it to an html document laika ignores the file.

Comment (Baccata) [reply docs/assets/sample_expectation_suite_output.html:17]: Mmm that's not ideal :/ , the point of this mdoc invocation is to exemplify the output of the framework. Not that it's critical to the docs but it's kinda nice to have.

Probs worth pinging some of the laïka maintainers for advice ?

Comment (CJSmith-0141) [reply docs/assets/sample_expectation_suite_output.html]: This was a problem with the ansi to html transformation making invalid html, fixed by making the html output valid.

see 8b1bca779916e43fedeb96f92398bb18f9cce0db

Comment (Baccata) [reply docs/assets/sample_expectation_suite_output.html]: @CJSmith-0141, would you mind just uploading a screenshot of the microsite on the PR description ?

**Review by Baccata — APPROVED** [review]

**Comment by CJSmith-0141** [comment]
Some screenshots

Landing page
 
Screenshot 2024-03-22 at 11 32 46 AM

Expectations page 
Screenshot 2024-03-22 at 11 33 19 AM

Install page 
Screenshot 2024-03-22 at 11 34 31 AM

Tracing failures (with report)
Screenshot 2024-03-22 at 11 38 05 AM

**Pull request merged**